src/commands/link.ts

```diff
@@ -0,0 +1,35 @@
+import { flags as oclifFlags } from '@oclif/command'
+
+import { CustomCommand } from '../oclif/CustomCommand'
+import appsLink from '../modules/apps/link'
+
+export default class Link extends CustomCommand {
+  static description = 'Start a development session for this app'
+
+  static examples = []
+
+  static flags = {
+    help: oclifFlags.help({ char: 'h' }),
+    clean: oclifFlags.boolean({ char: 'c', description: 'Clean builder cache', default: false }),
+    setup: oclifFlags.boolean({
+      char: 's',
+      description: 'Do not add app dependencies to package.json and do not run Yarn',
```

Comment:
Is this description right?
```suggestion
      description: 'Setup typings before linking [see vtex setup --help]',
```